### PR TITLE
Fix weight initialization on neural_style_tutorial

### LINKED ISSUE #2672: [BUG] - Weight initialization on neural_style
### Add Link

[neural_style_tutorial](https://pytorch.org/tutorials/advanced/neural_style_tutorial.html)

### Describe the bug

To produce
```bash
python neural_style_tutorial.py
```

This line causes this warning
```python
cnn = models.vgg19(pretrained=True).features.eval()
```

>UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(

>UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=VGG19_Weights.IMAGENET1K_V1`. You can also use `weights=VGG19_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)

### Describe your environment

Platform: macOS
CUDA: no
PyTorch version: 2.1.0

cc @sekyondaMeta @svekars @carljparker @NicolasHug @kit1980 @subramen

### PR DESCRIPTION
**Pull request by alperenunlu**
Fixes #2672

## Checklist

- [x] The issue that is being fixed is referred in the description
- [x] Only one issue is addressed in this pull request
- [x] Labels from the issue that this PR is fixing are added to this pull request
- [x] No unnecessary issues are included into this pull request.

cc @sekyondaMeta @svekars @carljparker @NicolasHug @kit1980 @subramen @albanD

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Thanks!
